Symfony/CS/Tests/ConfigurationResolverTest.php

```diff
@@ -257,9 +287,20 @@ public function testResolveProgressWithNegativeConfigAndPositiveOption()
 
     public function testResolveProgressWithNegativeConfigAndNegativeOption()
     {
-        $this->config->hideProgress(false);
+        $config = $this->getMock('Symfony\CS\Config\Config');
```

Comment:
Mock is already created in `setUp` under `$this->config`.
No need to create new one and inject it to `$this->resolver`.